pkg/storaged/filesystem/utils.jsx

```diff
@@ -178,3 +178,16 @@ export const MountPoint = ({ fstab_config, forced_options, backing_block, conten
             { extra_text }
         </>);
 };
+
+export const mount_point_text = (mount_point, mounted) => {
+    let mp_text;
+    if (mount_point) {
+        mp_text = client.strip_mount_point_prefix(mount_point);
+        if (mp_text == false)
+            return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19824-20240108-135624-fcdf3177-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)